libyul/AsmAnalysis.cpp

```diff
@@ -234,7 +234,7 @@ void AsmAnalyzer::operator()(VariableDeclaration const& _varDecl)
 				m_errorReporter.typeError(
 					3947_error,
 					variable.location,
-					"Assigning value of type \"" + givenType.str() + "\" to variable of type \"" + variable.type.str() + "."
+					"Assigning value of type \"" + givenType.str() + "\" to variable of type \"" + variable.type.str() + "\"."
```

Comment:
`escapeAndQuoteString` could be used here instead

Reply:
Yeah... on the other hand we abandoned the PR trying to use quoting functions everywhere...

Reply:
Ah did we? I thought it went through. Nevermind then.